website/source/docs/providers/datadog/r/user.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: "datadog"
+page_title: "Datadog: datadog_user"
+sidebar_current: "docs-datadog-resource-user"
+description: |-
+  Provides a Datadog user resource. This can be used to create and manage users.
+---
+
+# datadog\_user
+
+Provides a Datadog user resource. This can be used to create and manage Datadog users.
+
+## Example Usage
+
+```
+# Create a new Datadog user
+resource "datadog_user" "foo" {
+  email  = "new@example.com"
+  handle = "new@example.com"
+  name   = "New User"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `disabled` - (Optional) Whether the user is disabled
+* `email` - (Required) Email address for user
+* `handle` - (Required) Datadog ID for user (generally same as email)
```

Comment:
According to DD API docs: `The user handle, must be a valid email.` Might be worth adding as I'm assuming they will return 400 if it does not look like an email address, and users might be confused if it fails.